services/api/routes/authenticated.js

```diff
@@ -279,15 +282,18 @@ var routes = [
           project: numericSchema
         },
         payload: {
-          title: Joi.string().max(250).required(),
-          thumbnail: Joi.object().optional()
+          title: Joi.string().max(250).optional(),
```

Comment:
This is the difference between create and update. Create must provide a title, and update could just be for a description, not requiring a title.